components/backup-stream/src/endpoint.rs

```diff
@@ -666,7 +667,7 @@ where
                     TimeStamp::new(rts),
                     // Add a service safe point for 10mins (2x the default flush interval).
                     // It would probably be safe.
-                    Duration::from_secs(600),
+                    Duration::from_secs(1800),
```

Comment:
this would be 30 minutes

Reply:
Yes, for the safety of checkpoint from being GC, modify it from 10 mins to 30 mins.